index.html

```diff
@@ -0,0 +1,183 @@
+      <article>
+        Where you want to go?<br><code>Bangladesh TM</code>
+      </article>
+    </section>
+
+    <section class="main-section" id="Country_USA">
+      <header>
+        <h1>Country USA</h1>
+      </header>
+      <article>
+        <p>
+          The U.S. is a country of 50 states covering a vast swath of North America, with Alaska in the northwest and
+          Hawaii extending the nation’s presence into the Pacific Ocean. Major Atlantic Coast cities are New York
+        </p>
+        <p>
+        <ul>
+          <li>Washington DC</li>
+          <li>California</li>
+          <li>Virginia</li>
+          <li>Las Vegas</li>
+        </ul>
+        </p>
+
+      </article>
+
+      <h4>Hello USA</h4>
+      <article>
+        Where you want to go?<br><code>USA TM</code>
+      </article>
+    </section>
+
+    <section class="main-section" id="Country_Canada">
+      <header>
+        <h1>Country Canada</h1>
+      </header>
+      <article>
+        <p>
+          Canada is a country in North America. Its ten provinces and three territories extend from the Atlantic Ocean
+          to the Pacific Ocean and northward into the Arctic Ocean, covering over 9.98 million square kilometres
+        </p>
+        <p>
+        <ul>
+          <li>Toronto</li>
+          <li>Alberta</li>
+          <li>British Columbia</li>
+          <li>Manitoba</li>
+        </ul>
+        </p>
+
+      </article>
+
+      <header>Hello Canada</header>
+      <article>
+        Where you want to go?<br><code>Canada TM</code>
+      </article>
+    </section>
+
+    <section class="main-section" id="Country_Dubai">
+      <header>
+        <h1>Country Dubai</h1>
+      </header>
+      <article>
+        <p>
+          Dubai is a city and emirate in the United Arab Emirates known for luxury shopping, ultramodern architecture
+          and a lively nightlife scene. Burj Khalifa, an 830m-tall tower, dominates the skyscraper-filled skyline.
+        </p>
+        <p>
+        <ul>
+          <li>UAE</li>
+          <li>Madina</li>
+          <li>Mecca</li>
+          <li>Al Aqsa</li>
+        </ul>
+        </p>
+
+      </article>
+
+      <header>Hello Dubai</header>
+      <article>
+        Where you want to go?<br><code>Dubai TM</code>
+      </article>
+    </section>
+  </main>
+  <nav id="navbar">
+    <header>
+      <h2>Country Documentation</h2>
+    </header>
+    <a href="#The_World" class="nav-link">The World</a><br>
+    <a href="#Country_Egypt" class="nav-link">Country Egypt</a><br>
+    <a href="#Country_Bangladesh" class="nav-link">Country Bangladesh</a><br>
+    <a href="#Country_USA" class="nav-link">Country USA</a><br>
+    <a href="#Country_Canada" class="nav-link">Country Canada</a><br>
+    <a href="#Country_Dubai" class="nav-link">Country Dubai</a><br>
+  </nav>
+
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+  <script src="js/index.js"></script>
```

Comment:
> * [ ] Kindly add the `javascript` file . 👍

Reply:
- [ ] On this part also, kindly ensure to provide the reason when the javascript file should be added. explain what could be the hinderances for missing it.